app/views/index/normal.phtml

```diff
@@ -19,8 +19,8 @@ if (!empty($this->entries)) {
 	$today = @strtotime('today');
 ?>
 
-<div id="stream" class="normal<?php echo $hidePosts ? ' hide_posts' : ''; ?>"><?php
-	?><div id="new-article">
+<div id="stream" class="normal<?php echo $hidePosts ? ' hide_posts' : ''; ?>">
+	<div id="new-article">
```

Comment:
This was actually on purpose, to avoid sending some blanks to the client.
It is mostly irrelevant here (probably better without, indeed), but can be advantageous in particular in some loops.

Reply:
Huh. I've never heard of anything like that before. I want to run by what I'm understanding from your explanation to make sure that I've got it right: When in a php loop, an empty tag like that would let there be a newline in the php code that wouldn't exist in the html that's sent to to client?

Reply:
Except for that restrictive "in a loop" clause, that's correct.